### PR TITLE
More fixes/enhancement

### PR DESCRIPTION
**Pull request by pvicente**
Output for html target:

```bash
13:20 $ make html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.6.6
making output directory...
loading pickled environment... not yet created
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 156 source files that are out of date
updating environment: 156 added, 0 changed, 0 removed
reading sources... [100%] videos                                                                                                                                                         
looking for now-outdated files... none found
pickling environment... done
checking consistency... /home/pedro/git_projects/conan.io/docs/howtos/use_gtest.rst: WARNING: document isn't included in any toctree
done
preparing documents... done
writing output... [100%] videos                                                                                                                                                          
/home/pedro/git_projects/conan.io/docs/integrations/android_studio.rst:90: WARNING: Could not lex literal_block as "groovy". Highlighting skipped.
generating indices... genindex
writing additional pages... search
copying images... [100%] integrations/../images/clion/configure_warning_info.png                                                                                                         
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded, 2 warnings.
Generating sitemap.xml in /home/pedro/git_projects/conan.io/docs/_build/html/sitemap.xml

Build finished. The HTML pages are in _build/html.
```

### CODE REVIEW

File: devtools/running_packages.rst [hunk 1]
```diff
@@ -192,12 +209,9 @@ Installing and running this package, can be done by any of the means presented a
 
 .. code-block:: bash
 
-    $ conan install HelloRun/0.1@user/testing -g virtualrunenv
+    $ conan install . -g virtualrunenv
```
Comment (memsharded) [inline devtools/running_packages.rst]: In this case it wants to show how to directly install packages, without a local conanfile

File: creating_packages/define_abi_compatibility.rst [hunk 2]
```diff
@@ -20,7 +20,7 @@ When this package is installed by a *conanfile.txt*, another package *conanfile.
 
 .. code-block:: bash
 
-    $ conan install MyLib/1.0@user/channel -s arch=x86_64 -s ...
+    $ conan install . MyLib/1.0@user/channel -s arch=x86_64 -s ...
```
Comment (danimtb) [inline creating_packages/define_abi_compatibility.rst]: is it `$ conan install MyLib/1.0@user/channel` or `$ conan install .` ?

File: creating_packages/define_abi_compatibility.rst [hunk 3]
```diff
@@ -39,7 +39,7 @@ If the package is installed again with different settings, for example, for 32bi
 
 .. code-block:: bash
 
-    $ conan install MyLib/1.0@user/channel -s arch=x86 -s ...
+    $ conan install . MyLib/1.0@user/channel -s arch=x86 -s ...
```
Comment (danimtb) [inline creating_packages/define_abi_compatibility.rst]: same here?

File: devtools/running_packages.rst [hunk 4]
```diff
@@ -62,28 +62,35 @@ We will get some files, that can be called to activate and deactivate such envir
...
+With the ``deploy()`` method, a package can specify which files and artifacts to copy to user space or to other
+locations in the system. Let's modify the example recipe adding the ``deploy()`` method:
 
 .. code-block:: python
 
     def deploy(self):
         self.copy("*", dst="bin", src="bin")
 
 
+And call ``conan create``
```
Comment (danimtb) [inline devtools/running_packages.rst]: ``$ conan create``

File: devtools/running_packages.rst [hunk 5]
```diff
@@ -119,6 +126,15 @@ the ``Consumer`` package is willing to execute the ``greet`` app while building
             with tools.environment_append(env.vars):
                 self.run("greet")
 
+Now run ``conan install`` and ``conan build`` for this consumer recipe:
```
Comment (danimtb) [inline devtools/running_packages.rst]: ``$ conan install`` and ``$ conan build``

File: devtools/running_packages.rst [hunk 6]
```diff
@@ -154,6 +170,7 @@ to greet executable:
 
 .. _repackage:
 
+
```
Comment (danimtb) [inline devtools/running_packages.rst]: new line not needed

File: creating_packages/define_abi_compatibility.rst (hunk 3 above)
Comment (memsharded) [reply creating_packages/define_abi_compatibility.rst]: True, you either install the path "." or the package "MyLib/1.0@user/channel"

File: creating_packages/define_abi_compatibility.rst (hunk 2 above)
Comment (memsharded) [reply creating_packages/define_abi_compatibility.rst]: True, you either install the path "." or the package "MyLib/1.0@user/channel"

**Pull request merged**